### PR TITLE
Obo Agboghidi DOM-1

### PR DESCRIPTION
**Pull request by oagboghi2**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -36,19 +36,73 @@ const siteContent = {
...
+// siteContent["nav"].forEach(element => {
+//   aTag;
+// });
+aTag[0].innerHTML = siteContent["nav"]["nav-item-1"]
+aTag[1].innerHTML = siteContent["nav"]["nav-item-2"]
+aTag[2].innerHTML = siteContent["nav"]["nav-item-3"]
+aTag[3].innerHTML = siteContent["nav"]["nav-item-4"]
+aTag[4].innerHTML = siteContent["nav"]["nav-item-5"]
+aTag[5].innerHTML = siteContent["nav"]["nav-item-6"]
+
```
Comment (JasonCorp84) [inline js/index.js]: works well, also it is a perfect spot for a for loop, or a higher order function. 
👍

File: js/index.js [hunk 2]
```diff
@@ -36,19 +36,73 @@ const siteContent = {
...
+
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
 
+//Image of code snippet
+let imag = document.getElementById("cta-img");
+imag.setAttribute('src', siteContent["cta"]["img-src"])
+
```
Comment (JasonCorp84) [inline js/index.js]: 👍

**Comment by JasonCorp84** [comment]
Obo,

## DOM I - JavaScript Content Management System
- [x ] Task 1 - Create selectors to point your data into elements
- [x ] Task 2 - Update the HTML with the JSON data
- [x ] Task 3 - Add new content
- [x ] Stretch Goal: update styles throughout
- [ ] Stretch Goal: integrate a button to update content

## Good Habits
- [x ] Pull Request title is your first and last name (add any notes in the comment field)
- [ ] Code has been committed regularly
- [ ] Commit comments are informative
- [x ] Code is organized in a logical way

Thank you for submitting your PR. Please also let's make sure that your commits are informative.
I am looking forward to seeing you every day like today. Keep up the good work!

**Comment by mixelpixel** [comment]
@JasonCorp84 I took out the extra space for the "checked" Task List syntax 😄

**Pull request closed without merging**